qrand.go

```diff
@@ -66,6 +68,25 @@ func Read(buf []byte) (n int, err error) {
 	return Reader.Read(buf)
 }
 
+type ΨSource struct{}
+
+var qSource ΨSource
+
+func (s *ΨSource) Seed(seed int64) {}
+
+func (s *ΨSource) Uint64() (value uint64) {
+	binary.Read(Reader, binary.BigEndian, &value)
+	return value
+}
+
+func (s *ΨSource) Int63() (value int64) {
+	return int64(s.Uint64() & ^uint64(1<<63))
+}
+
+func New() *rand.Rand {
```

Comment:
Actually, thinking about it, does it really make sense to import `rand` just to call `rand.New` on `Source` and return a `rand.Rand`? Probably not. The user would just write instead:

```go
var random = rand.New(qrand.Source{})
```